doc/source/tutorial/optimize.rst

```diff
@@ -1467,52 +1472,47 @@ where
     \end{bmatrix}
     \end{equation*}
 
-The last inequality constraint :math:`x_i \geq 0` is minimum value constraint for all decision variables, which can be
-applied as the ``bounds`` argument of :func:`linprog`. As you can see the API doc :func:`linprog`, the default values of
-``bound`` argument is ``(0, None)`` that means all decision variables are non-negative. So, we don't need to do anything
-the last inequality constraint in this case.
+Lastly, let's consider the minimum and maximum inequality constraints for each decision variable, which is known as
+"box constraint". These constraints can be applied as the ``bounds`` argument of :func:`linprog`.
+As you can see the API doc :func:`linprog`, the default values of ``bounds`` argument is ``(0, None)`` that means
+all decision variables are non-negative. Using None indicates that there is no bound. So, we need to set each bound
+as ``bounds`` argument in this case.
 
-Finally, we can solve the desired problem using the mathematical converts and :func:`linprog`.
+Finally, we can solve the transformed problem using :func:`linprog`.
 
 ::
 
-    from scipy.optimize import linprog
-    import numpy as np
-
-    c = np.array([-29.0, -45.0, 0.0, 0.0])
-    A_ub = np.array([[1.0, -1.0, -3.0, 0.0],
-                    [-2.0, 3.0, 7.0, -3.0]])
-    b_ub = np.array([[5.0],
-                    [-10.0]])
-    A_eq = np.array([[2.0, 8.0, 1.0, 0.0],
-                     [4.0, 4.0, 0.0, 1.0]])
-    b_eq = np.array([[60.0],
-                    [60.0]])
-
-    result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq)
+    >>> from scipy.optimize import linprog
+    >>> import numpy as np
+    >>> c = np.array([-29.0, -45.0, 0.0, 0.0])
+    >>> A_ub = np.array([[1.0, -1.0, -3.0, 0.0],
+    ...                [-2.0, 3.0, 7.0, -3.0]])
+    >>> b_ub = np.array([[5.0],
+    ...                [-10.0]])
+    >>> A_eq = np.array([[2.0, 8.0, 1.0, 0.0],
+    ...                 [4.0, 4.0, 0.0, 1.0]])
+    >>> b_eq = np.array([[60.0],
+    ...                [60.0]])
+    >>> x0_bounds = (0, None)
+    >>> x1_bounds = (0, 5.0)
+    >>> x2_bounds = (None, 0.5)
+    >>> x3_bounds = (-3.0, 0)
+    >>> bounds = [x0_bounds, x1_bounds, x2_bounds, x3_bounds]
+    >>> result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq, bounds=bounds)
 
 The result is
 
 ::
 
     >>> print(result)
-    con: array([3.07489501e-09, 3.19440119e-09])
-    fun: -500.79999997382595
-    message: 'Optimization terminated successfully.'
-    nit: 5
-    slack: array([ 1.0000000e+00, -1.0003518e-09])
-    status: 0
-    success: True
-    x: array([9.2000000e+00, 5.2000000e+00, 4.3425461e-11, 2.4000000e+00])
-
-::
-
-    >>> [print("x_", i, ":", np.round(result.x[i], 2)) for i in range(len(result.x))]
-    x_ 0 : 9.2
-    x_ 1 : 5.2
-    x_ 2 : 0.0
-    x_ 3 : 2.4
-
+        con: array([42.58855175, 45.54023356])
```

Comment:
```suggestion
        con: array([42.58855175, 45.54023356])  # may vary
```
This is why Travis is failing. It performs a docstring test, so you need to indicate that the numbers can change.

Reply:
I fixed.